nixos/modules/services/monitoring/karma.nix

```diff
@@ -0,0 +1,124 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  cfg = config.services.karma;
+  yaml = pkgs.formats.yaml { };
+in
+{
+  options.services.karma = {
+    enable = mkEnableOption (mdDoc "the Karma dashboard service.");
```

Comment:
`mkEnableOption` includes a trailing period.

Reply:
Didn't know that, pushed the change